packages/api/src/command/medical/facility/create-facility.ts

```diff
@@ -1,17 +1,34 @@
 import { uuidv7 } from "@metriport/core/util/uuid-v7";
 import { FacilityData } from "../../../domain/medical/facility";
-import { FacilityModel } from "../../../models/medical/facility";
+import { FacilityModel, FacilityType } from "../../../models/medical/facility";
 
 export const createFacility = async ({
   cxId,
   data,
+  type = FacilityType.initiatorAndResponder,
+  cqOBOActive = false,
+  cwOBOActive = false,
+  cqOBOOID = "",
+  cwOBOOID = "",
```

Comment:
nit: make it fully camelCase: `cqOboOid`

Me and Ramil have been talking about this [here](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#65562fe296794944a864eebd6d7a2e93)

Also, let's avoid using empty strings for default values, please - [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#cff793e9f7f5445fb075c82490040bc1)